src/VisualStudio/IntegrationTest/IntegrationTests/Typescript/TypescriptCompletionTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using ProjectUtils = Microsoft.VisualStudio.IntegrationTest.Utilities.Common.ProjectUtils;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Typescript
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class TypescriptCompletionTest : AbstractEditorTest
+    {
+        public TypescriptCompletionTest(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(TypescriptCompletionTest))
+        {
+            var project = new ProjectUtils.Project(ProjectName);
+            VisualStudio.SolutionExplorer.AddFile(project, "Lib.ts", contents:
+@"export function foo(): string {
+    return ""Hello World""
+}");
+            VisualStudio.SolutionExplorer.AddFile(project, "File1.ts");
+            VisualStudio.SolutionExplorer.OpenFile(project, "File1.ts");
+
+            // Wait for current file to become part of the typescript project
```

Comment:
I wouldn't expect this wait to actually work for that, because they're doing their own async stuff? @paulvanbrenk or @minestarks?